Tools/ardupilotwaf/chibios.py

```diff
@@ -205,7 +205,7 @@ def run(self):
         desc_len = 16
         crc1 = to_unsigned(crc32(bytearray(img[:offset])))
         crc2 = to_unsigned(crc32(bytearray(img[offset+desc_len:])))
-        githash = to_unsigned(int('0x' + os.environ.get('GIT_VERSION', self.generator.bld.git_head_hash(short=True)),16))
+        githash = to_unsigned(int('0x' + os.environ.get('GIT_VERSION') if 'GIT_VERSION' in os.environ else self.generator.bld.git_head_hash(short=True),16))
```

Comment:
We don't tend to use this if/else format; please split this across multiple statements.